doc: Governance changes for Collaboration Network (#547)
openjs-foundation/cross-project-council: README.md

* [Fastify](https://fastify.io)
* [nvm](http://nvm.sh/)

### OpenJS Collaboration spaces

The following are official OpenJS Collaboration spaces. If you are interested in fostering collaboration with the 
support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression]

> 

Reviewer: Should there be a note here that there are none? It seems weird for it to mention `The following are official OpenJS Collaboration spaces` but then have nothing blow

Author: @MylesBorins good catch, added.

Author: added

> support of the OpenJS foundation through a collaboration space, plear read our [Collaboration Space Progression](./proposals/stage-0/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md) document, or contact any CPC member with questions.

Reviewer 2: > plear read our

please read our

> support of the OpenJS foundation through a collaboration space, plear read our [Collaboration Space Progression](./proposals/stage-0/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md) document, or contact any CPC member with questions.

Author: ```suggestion
support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression]
```

* [Fastify](https://fastify.io)
* [nvm](http://nvm.sh/)

### OpenJS Collaboration spaces

The following are official OpenJS Collaboration spaces. If you are interested in fostering collaboration with the 

> support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression]

Reviewer 3: It seems this link is blank.

Reviewer 4: I think it's supposed to be: `[Collaboration Space Progression](./COLLABORATION_SPACE_PROGRESSION.md)`

> support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression]

Author: ```suggestion
support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression](./COLLABORATION_SPACE_PROGRESSION.md)
```

- Christopher Hiller (@boneskull, IBM)
- Tobie Langel (@tobie, UnlockOpen)

#### Collaboration space representatives

> Members of the Collaboration spaces at the `Core` stage may nominate a candiate for one of the voting seats on the CPC which represents the Collaboration spaces. Currently there are no spaces at the `Core` stage and therefore no representative.

Reviewer: What is the `Core` phase? Where is the documentation for this?

Author: These are documented here https://github.com/openjs-foundation/cross-project-council/blob/master/proposals/stage-0/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md#core-collaboration-spaces

### Project Resources & Travel Fund

OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md). Details can be found in the [/project-resources](./project-resources) directory.

> OpenJS Foundation Collaboration spaces are able to take advantage of serveral services and benefits as outlined in [Collaboration Space Progression](./proposals/stage-0/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md). 

Reviewer 2: > serveral services and benefits

several services and benefits

> OpenJS Foundation Collaboration spaces are able to take advantage of serveral services and benefits as outlined in [Collaboration Space Progression](./proposals/stage-0/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md). 

Author: ```suggestion
OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [Collaboration 
```

### Project Resources & Travel Fund

OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md). Details can be found in the [/project-resources](./project-resources) directory.

> OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [Collaboration 

Reviewer 5: This sentence doesn't seem to be complete. I am not entirely sure which section it refers to.

Author: @christian-bromann thanks for catching that. Fixed although the link won't work until https://github.com/openjs-foundation/cross-project-council/pull/595 lands

> OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [Collaboration 

Author: ```suggestion
OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [OLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md).
```

> OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [OLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md).

Reviewer 4: Looks like a small typo: the link to the `COLLABORATION_NETWORK.md` doc is missing a "C" at the beginning (currently: `...OLLABORATION_NETW...`)

> OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [OLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md).

Author: ```suggestion
OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [COLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md).
```

For any other topics which aren't covered above, please email
[operations@openjsf.org](mailto:operations@openjsf.org).

> 

Reviewer: Is there no longer a newline at the end of the document?

Author: Looks like there is still one.
